src/main/kotlin/no/nav/familie/baks/mottak/søknad/PdfService.kt

```diff
@@ -1,110 +1,20 @@
 package no.nav.familie.baks.mottak.søknad
 
-import no.nav.familie.baks.mottak.integrasjoner.PdfClient
 import no.nav.familie.baks.mottak.søknad.barnetrygd.domene.DBBarnetrygdSøknad
-import no.nav.familie.baks.mottak.søknad.barnetrygd.domene.SøknadV7
-import no.nav.familie.baks.mottak.søknad.barnetrygd.domene.SøknadV8
 import no.nav.familie.baks.mottak.søknad.barnetrygd.domene.VersjonertSøknad
 import no.nav.familie.baks.mottak.søknad.kontantstøtte.domene.DBKontantstøtteSøknad
-import no.nav.familie.baks.mottak.søknad.kontantstøtte.domene.KontantstøtteSøknadV3
-import no.nav.familie.baks.mottak.søknad.kontantstøtte.domene.KontantstøtteSøknadV4
 import no.nav.familie.baks.mottak.søknad.kontantstøtte.domene.VersjonertKontantstøtteSøknad
-import no.nav.familie.kontrakter.ba.søknad.v4.Søknadstype
-import org.slf4j.LoggerFactory
-import org.springframework.stereotype.Service
-import java.time.LocalDateTime
-import java.time.format.DateTimeFormatter
-import java.time.format.FormatStyle
-import java.util.Locale
 
-@Service
-class PdfService(
-    private val pdfClient: PdfClient,
-    private val søknadSpråkvelgerService: SøknadSpråkvelgerService,
-) {
+interface PdfService {
```

Comment:
Har ikke så mye å si siden koden etterhvert skal slettes. GammelPdfService og NyPdfService er helt like utenom opprettting av pdf. Så jeg hadde nok i stedet for lagd ett interface for oppretting av pdf i stedet for dette interfacet.

PdfService - Interface
	lagPdf(templatename, inputData)
	
Somme hadde hatt 2 ulike implementasjoner DokgenPdfService og FamilieDokumentPdfService. Så hadde man slyttet Ny og Gammel